ctapipe/image/muon/intensity_fitter.py

```diff
@@ -467,7 +472,7 @@ class MuonIntensityFitter(TelescopeComponent):
     ).tag(config=True)
 
     def __call__(
-        self, tel_id, center_x, center_y, radius, image, pedestal,
+        self, tel_id, center_x, center_y, radius, image, pedestal, mask
```

Comment:
This makes `mask` a required argument. This function is used in the muon reconstruction tool, so this breaks it (and this is also why the tests fail).

Adding a `mask=None` default and making it optional would be best, I think.